indigo_app/static/javascript/indigo/views/work_amendments.js

```diff
@@ -20,7 +20,10 @@
     addAmendment: function(e) {
       e.preventDefault();
 
-      var chooser = new Indigo.WorkChooserView({country: this.model.get('country')}),
+      var chooser = new Indigo.WorkChooserView({
+            country: this.model.get('country'),
+            locality: this.model.get('locality'),
```

Comment:
```suggestion
            locality: this.model.get('locality') || "-",
```
this will select (none) for locality when on country level